lib/matplotlib/tests/test_transforms.py

```diff
@@ -535,6 +536,21 @@ def test_nan_overlap():
     assert not a.overlaps(b)
 
 
+def test_transform_angles():
+    t = mtrans.Affine2D()
+    angles = np.array([20, 45, 60])
+    points = np.array([[0, 0], [1, 1], [2, 2]])
+
+    new_angles = t.transform_angles(angles, points)
+    assert_array_almost_equal(angles, new_angles)
+
+    with assert_raises(ValueError) as cm:
```

Comment:
for future coders, could you add a comment explaining the error you are triggering?